modules/rosa-unlinking-and-deleting-ocm-and-user-iam-roles.adoc

```diff
@@ -37,13 +63,24 @@ ROLE NAME                           ROLE ARN
 ManagedOpenShift-OCM-Role-<red_hat_organization_external_id>  arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-OCM-Role-<red_hat_organization_external_id>  Yes     Yes
 ----
 +
-.. If your {cluster-manager} IAM role is listed as linked in the output of the preceding command, unlink the role from your Red Hat organization:
+endif::hcp[]
+ifdef::hcp[]
+.Example output
+[source,terminal]
+----
+I: Fetching ocm roles
+ROLE NAME                           ROLE ARN                                                           LINKED  ADMIN  AWS Managed
+ManagedOpenShift-OCM-Role-<red_hat_organization_external_id>  arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-OCM-Role-<red_hat_organization_external_id>  Yes      Yes     Yes
```

Comment:
In my browser, LINKED, ADMIN, AWS, and Managed aren't lining up with the `Yes` columns below. 
![image](https://github.com/openshift/openshift-docs/assets/82119979/e7616e1e-bcde-41f5-bbc6-a32f72436517)